packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.tsx

```diff
@@ -1245,6 +1237,39 @@ class ViewportPositioner extends Foundation<
                 return ViewportPositionerVerticalPositionLabel.undefined;
         }
     };
+
+    /**
+     * Gets the uninitialized state
+     */
+    private getInitialState = (): ViewportPositionerState => {
```

Comment:
This is somewhat of a nit but I think important. While this function is descriptive of what is happening, there previously was a pattern of getInitialState in react and I'd like to avoid confusion with it. Can we modify this name slightly to do so? Perhaps `generateInitialState` or something close but not the same as the above? :)

Reply:
generateInitialState it is.